Gemfile

```diff
@@ -84,7 +84,7 @@ eval_gemfile("./Gemfile.local") if File.exist?("./Gemfile.local")
 #
 # We copy (and overwrite) these files every time "bundle <exec|install>" is
 # executed, just in case they have changed.
-if RUBY_PLATFORM =~ /mswin|mingw|windows/
+if RUBY_PLATFORM.match?(/mswin|mingw|windows/)
```

Comment:
String#match? was added in ruby 2.4, does chef support anything older than that?

Reply:
We're 2.6+ now. I've been waiting a while to use this, but the rubocop autocorrect before destroyed code.